http/src/test/resources/reference.conf

```diff
@@ -101,6 +100,11 @@ monitor {
   pollPeriod = 1 second
   maxRetries = -1  # means retry forever
   recreateCluster = true
+  creatingTimeLimit = 15 seconds
+  startingTimeLimit = 15 seconds
+  stoppingTimeLimit = 15 seconds
+  deletingTimeLimit = 15 seconds
+  updatingTimeLimit = 15 seconds
```

Comment:
Any downside to lowering these limits so tests can finish more quickly?

Reply:
when it is too low it has the potential to conflict with tests that use the cluster monitor in a hard-to-track-down way